src/borg/archive.py

```diff
@@ -91,6 +91,7 @@ def show_progress(self, item=None, final=False, stream=None, dt=None):
                 space = columns - swidth(msg)
                 if space < swidth('...') + swidth(path):
                     path = '%s...%s' % (path[:(space // 2) - swidth('...')], path[-space // 2:])
+                space -= swidth(path)
                 msg += "{0:<{space}}".format(path, space=space)
```

Comment:
that doesn't look correct.

"<N" means "left-align in a N spaces wide field", not "add N spaces after the argument".

https://pyformat.info/#string_pad_align